From 41a789949c1f6cf4ad50f6ece4566773bdd8e7f2 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Fri, 9 Dec 2022 21:08:39 +0100
Subject: [PATCH] Omit git version retrieval

---
 meson.build | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/meson.build b/meson.build
index 15fd5cd..05e162c 100644
--- a/meson.build
+++ b/meson.build
@@ -6,10 +6,6 @@ project('gcab', 'c',
 )
 
 git_version = []
-git = find_program('git', required: false)
-if git.found()
-  git_version = run_command(git, 'describe', '--abbrev=4', '--dirty', check: true).stdout().strip().split('-')
-endif
 
 # libtool versioning
 lt_current = 2
-- 
2.37.1 (Apple Git-137.1)

